extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/ETC1Compressor.java

```diff
@@ -58,17 +63,31 @@ protected void processDir (Entry entryDir, ArrayList<Entry> value) throws Except
 					throw new Exception("Couldn't create output directory '" + entryDir.outputDir + "'");
 			}
 		}
+
+		public void setTransparentColor (Color transparentColor) {
+			if (transparentColor == null) {
+				this.transparentColor.set(1, 0, 1, 1);
+			} else {
+				this.transparentColor.set(transparentColor);
+			}
+		}
 	}
 
-	public static void process (String inputDirectory, String outputDirectory, boolean recursive, boolean flatten)
-		throws Exception {
+	public static void process (String inputDirectory, String outputDirectory, boolean recursive, boolean flatten,
+		Color transparentColor) throws Exception {
 		GdxNativesLoader.load();

```

Comment:
@Darkyenus and here we don't.
Java Rule #44: "Avoid using more than one method/constructor argument whenever is possible."